parts/linux/cloud-init/artifacts/components.json

```diff
@@ -510,6 +510,20 @@
           }
         }
       }
+    },
+    {
+      "name": "oras",
```

Comment:
I remember you said oras is needed for another component to be downloaded? If that is the case, oras needs to be in the first place of this list, until we support a new key "dependencies".

Reply:
oh interesting, I thought it needed to be first in the install_dependencies loop. But it should be at the top of components.json as well?

Reply:
The loop doesn't matter actually. (Did I say we should modify the loop? lol)
The list in components.json controls which should be installed first. Make sense?